checkout-v3/authenticated/seamless-view.md

```diff
@@ -178,6 +178,12 @@ Once the payer has completed the purchase, you can perform a GET towards the
 You can read about the different [Seamless View Events][seamless-view-events] in
 the feature section.
 
+{% include alert.html type="success" icon="info" header="`onPaid` Event"
+body="When integrating Seamless View, we strongly recommend that you implement
+the `onPaid` event for the best possible setup. Even with this implemented, you
+need to check the payment status towards our APIs, as the payer can change this
+in the browser fast." %}
```

Comment:
Make a new include called `alert-onpaid-event.md` instead to avoid duplication?

Reply:
Okidoke!

Reply:
I took it in another direction, and also included dx-1709.